build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019 as downloader
+ADD https://aka.ms/vs/17/release/vc_redist.x64.exe /
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+# Set working directory
+WORKDIR c:\\
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+COPY --from=downloader /vc_redist.x64.exe /
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# Run the installation in PowerShell
+
+RUN powershell -Command "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract Open vSwitch
+ADD https://downloads.antrea.io/ovs/ovs-2.15.2-antrea.0-win64.zip .
```

Comment:
It's better to set this ovs link in build/images/deps rather than using hardcoded ovs link in dockerfile, please also update the Makefile.

Reply:
Also, this windows ovs image should be imported into build/images/base-windows/dockerfile

Reply:
> Also, this windows ovs image should be imported into build/images/base-windows/dockerfile

I think, did u mean build/images/ovs, as ovs images and their docker file belong here, also we need to make sure we push it to docker private repo !

Reply:
I mean the new base-windows image should involve win-ovs image just as what we did in linux base image.https://github.com/antrea-io/antrea/blob/779940743c6046ac63e7866903187a4f97d3adaa/build/images/base/Dockerfile#L40

We can set NO_PULL=1 and building new base-windows image before testing, then new agent image can be built without pulling base-windows image from dockerhub. https://github.com/antrea-io/antrea/blob/779940743c6046ac63e7866903187a4f97d3adaa/Makefile#L372

Reply:
this folder build/images/ovs is meant for building new ovs version, so we might need to shift this Dockerfile to antrea/build/images/base-windows folder, and we might not like to  use existing dockerfile build/images/base-windows/Dockerfile(it contains rancher wins which is not required for containerd case), we can have this new dockerfile dockerfile.windows for windows ovs image !

Reply:
Also in linux case, ovs base image exists in dockerhub registry, but in windows case ovs based image does not exist in dockerhub registry, so we so we can not pull it from dockerhub but it has to be downloaded from existing s3 bucket where windows ovs zip file reside

Reply:
> we can have this new dockerfile dockerfile.windows for windows ovs image

It will bring two different windows images for docker and containerd, If rancher wins doesn't block the containerd workflow I think we can keep it to maintain the same image for different runtimes.

> Also in linux case, ovs base image exists in dockerhub registry, but in windows case ovs based image does not exist in dockerhub registry

We can update /hack/build-antrea-windows-all.sh to add building and pushing windows ovs base image logic.

Reply:
I think we can leverage the existing Dockerfile in antrea/build/images/base-windows as currently it is already used to build multiple images based on a single dockerfile and as u mentioned, can make use of build-antrea-windows-all.sh to generate another image for ovs. We can take a call on same.